src/components/MoneyRaised.js

```diff
@@ -10,6 +10,7 @@ const MONEY_RAISED = 1057100.0
 const MONEY_RAISED_UPDATE_TIME = moment('2021-01-07T16:30:00.000Z')
 const DOLLARS_PER_DAY_RATE = 700.0
 
+// todo: Turn this into a functional component so we can just reuse the hook here
```

Comment:
In a later refactor, we could just remove this component altogether. Now that we have the hook, this component is just barely useful in presentation. (The hook could even return the USD currency-formatted value, if we want.)

Reply:
Yep, that's fair. It'd be pretty trivial to just call the hooks at all the callistes.

Reply:
We'd still only want to use the hook at the top level in that case, unless we use context for the money raised value. Otherwise, we'd end up with out-of-sync values because each hook manages its own state and Javascript intervals are not precise.

Reply:
Sorry, I meant in other routes :)